### ISSUE TITLE
Restore the splatting feature of the Polyhedron demo

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal#166]
The pull request #154 will disable the splatting feature of the Polyhedron demo, because the current `GlSplat` code is incompatible with recent OpenGL standards.

This issue is to remember we should restore that feature.

**Comment by sloriot** [comment]
Removed by commit 3785112

**Comment by maxGimeno** [comment]
I made two branches implementing a different version of Splatting : cgal-public-dev/CGAL-Qt5_support-Splatting-GF  and   cgal-public-dev/CGAL-Qt5_support-Splatting-Perf-GF.

 The first one uses instanced rendering to draw discs but it is not quick enough to work well on big items.

 The second one uses a modified version of GlSplat that do not use glew anymore. It is working, but it still uses the OpenGL direct mode (like libQGLViewer) for now. I don't have the time to finish the upgrading to the "not direct" mode before I leave tonight, so I just pushed it like it is, as it works, even if it is not really "clean".
